src/services/utilities.ts

```diff
@@ -565,6 +565,7 @@ namespace ts {
 
     export function isStringOrRegularExpressionOrTemplateLiteral(kind: SyntaxKind): boolean {
         if (kind === SyntaxKind.StringLiteral
+            || kind === SyntaxKind.StringLiteralType

```

Comment:
This `kind === SyntaxKind.StringLiteral || kind === SyntaxKind.StringLiteralType` thing comes up quite a few times. Can you add some sort of helper function sort of like this code is sitting within?